prow/git/v2/interactor.go

```diff
@@ -80,6 +80,20 @@ type cloner interface {
 	Clone(from string) error
 }
 
+// MergeOpt holds information of a commit message
+type MergeOpt struct {
```

Comment:
No, it holds options for Merges which happens to only be the commit message today

Reply:
addressed